packages/nextjs/components/scaffold-eth/Input/InputBase.tsx

```diff
@@ -35,7 +35,7 @@ export const InputBase = <T extends { toString: () => string } | undefined = str
     <div className={`flex border-2 border-base-300 bg-base-200 rounded-full text-accent ${modifier}`}>
       {prefix}
       <input
-        className="input input-ghost focus:outline-none focus:bg-transparent focus:text-secondary-content h-[2.2rem] min-h-[2.2rem] px-4 border w-full font-medium placeholder:text-accent/50 text-secondary-content"
+        className="input input-ghost focus:outline-none focus:bg-transparent focus:text-gray-400 h-[2.2rem] min-h-[2.2rem] px-4 border w-full font-medium placeholder:text-accent/50 text-gray-400"
```

Comment:
I think should put it back to `text-secondary-content`  🙌 : 

with `text-gray-400` : 
![Screenshot 2023-09-30 at 8 35 03 PM](https://github.com/scaffold-eth/se-2-challenges/assets/80153681/2bb63f92-4676-451c-b407-11aaa4205d7d)